packages/convex-helpers/react/sessions.ts

```diff
@@ -124,10 +122,11 @@ export const SessionProvider: React.FC<{
   );
   const value = useMemo(
     () => ({
-      sessionId: initial && ssrFriendly ? SSR_DEFAULT : sessionId,
+      sessionId:
+        ssrFriendly && initial ? SSR_DEFAULT : sessionId || SSR_DEFAULT,
```

Comment:
Maybe
```suggestion
        ssrFriendly && initial ? SSR_DEFAULT : sessionId ?? SSR_DEFAULT,
```

Reply:
Yeah I thought about that too. I'm thinking an empty string is also a bad value, if they had manually edited their localStorage or something. If they have a custom session ID generator maybe they want the empty string sentinel value, but I'll just wait for someone to complain on that I think